tests/test_plugin_internallinks.py

```diff
@@ -0,0 +1,112 @@
+from unittest.mock import Mock, patch
```

Comment:
Nice job following the existing patterns with the testing. This is all great, nice work!